server/api.go

```diff
@@ -157,11 +159,33 @@ func apiActionFull(baseURL string, a *action.Action) ActionFull {
 	jsonschema := a.JSONSchema()
 	jsonschema.ID = fmt.Sprintf("%s/actions/%s/schema.json", baseURL, url.QueryEscape(a.ID))
 	def := a.ActionDef()
+
+
+	var resultMap map[string]interface{}
+
+  yamlData, err := os.ReadFile(fmt.Sprintf("%s/ui-schema.yaml", a.Dir()))
```

Comment:
fix indent please, command `make lint` will do it for you

Reply:
fixed